plugins/ntuplizer.cc

```diff
@@ -431,6 +505,105 @@ void ntuplizer::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
     // lowPtElecs.avgClusterZ.push_back(tcz/nbsc_seed);
    }
 
+   for (int i=0; i<nLowPtElectron; i++){
+    //Gets first electron
+    std::cout << "Electron" << std::endl;
+    std::cout << i << std::endl;
+    Lxy_SV = -99;
+    normalizedChi2 = 0;
+    //hasValidVertex = false;
+    for (int j=i+1; j<nLowPtElectron; j++){
+      std::cout << "Paied with Electron" << std::endl;
+      std::cout << j << std::endl;
+      //isEE = false;
+      double charge1 = lowPtElectrons->at(i).charge();
+      double charge2 = lowPtElectrons->at(j).charge();
+      if (charge1*charge2>0){
+        std::cout << "Invalid Pair" << std::endl;
```

Comment:
I would **comment** the printing, because when running the code over large datasets it will make it slow.